config.md

```diff
@@ -247,7 +247,7 @@ Note: Any OPTIONAL field MAY also be set to null, which is equivalent to being a
        This field is used to mark if the history item created a filesystem diff.
        It is set to true if this history item doesn't correspond to an actual layer in the rootfs section (for example, Dockerfile's [ENV](https://docs.docker.com/engine/reference/builder/#/env) command results in no change to the filesystem).
 
-Any extra fields in the Image JSON struct are considered implementation specific and MUST be ignored by any implementations which are unable to interpret them.
+Any extra fields in the Image JSON struct are considered implementation specific and MUST NOT generate an error by any implementations which are unable to interpret them.
```

Comment:
This is specific to _registry_ implementations, right? (as opposed to "client implementations" which feels included in that "any")

In other words, the intent is that we expect Docker Hub and friends to accept Helm charts, but it's still 100% reasonable and expected for the Docker CLI/Engine to bail with a helpful error if you try to `docker run` a Helm chart. :sweat_smile:

Reply:
Yeah, hm.

An example of a client that's in-the-middle is, e.g., `oras copy`. I expect ORAS to faithfully copy manifests and blobs from one registry to another. But I suppose that's a "MAY".